alacritty/src/display.rs

```diff
@@ -253,8 +254,10 @@ impl Display {
         #[cfg(target_os = "macos")]
         set_font_smoothing(config.ui_config.font.use_thin_strokes());
 
-        #[cfg(not(any(target_os = "macos", windows)))]
+        #[cfg(all(feature = "x11", not(any(target_os = "macos", windows))))]
         let is_x11 = event_loop.is_x11();
+        #[cfg(not(any(feature = "x11", target_os = "macos", windows)))]
+        let is_x11 = false;
```

Comment:
Does this call not work without the backend enabled in glutin? Otherwise it's probably not worth it gating it just for the negligible performance difference.

Reply:
it doesn't work, there's `is_wayland` though.

Reply:
And that always works? I don't mind either way since it wouldn't change anything for us. Or are you saying we could call is_wayland instead? Which would be pointless.

Reply:
No, it doesn't always work, it just that knowing that it's a case when you have wayland feature we can call this function, but you can make one statement out of this.

Reply:
How could we make one statement out of it? We'd need another one for `is_x11 = true` then wouldn't we?

Reply:
You can't, typo.